### ISSUE TITLE
Neo 3.1.3 doesn't work with PostgreSQL

### ISSUE THREAD

**Issue by boboldehampsink** [spicywebau/craft-neo#609]
It tries to set FALSE to an integer field

```
> applying existing project config ... Exception: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type integer: "f"
--
  | The SQL being executed was: SELECT "elementId"
  | FROM "structureelements"
  | WHERE ("structureId"=FALSE) AND ("elementId" IS NOT NULL) (/app/user/vendor/yiisoft/yii2/db/Schema.php:676)
  | #0 /app/user/vendor/yiisoft/yii2/db/Command.php(1307): yii\db\Schema->convertException(Object(PDOException), 'SELECT "element...')
  | #1 /app/user/vendor/yiisoft/yii2/db/Command.php(1168): yii\db\Command->internalExecute('SELECT "element...')
  | #2 /app/user/vendor/yiisoft/yii2/db/Command.php(453): yii\db\Command->queryInternal('fetchAll', 7)
  | #3 /app/user/vendor/yiisoft/yii2/db/Query.php(320): yii\db\Command->queryColumn()
  | #4 /app/user/vendor/craftcms/cms/src/db/Query.php(214): yii\db\Query->column(NULL)
  | #5 /app/user/vendor/spicyweb/craft-neo/src/services/Fields.php(289): craft\db\Query->column()
  | #6 /app/user/vendor/spicyweb/craft-neo/src/Field.php(811): benf\neo\services\Fields->saveValue(Object(benf\neo\Field), Object(craft\elements\GlobalSet))
  | #7 /app/user/vendor/craftcms/cms/src/base/Element.php(4535): benf\neo\Field->afterElementPropagate(Object(craft\elements\GlobalSet), true)
  | #8 /app/user/vendor/craftcms/cms/src/services/Elements.php(2733): craft\base\Element->afterPropagate(true)
  | #9 /app/user/vendor/craftcms/cms/src/services/Elements.php(766): craft\services\Elements->_saveElementInternal(Object(craft\elements\GlobalSet), false, true, true)
  | #10 /app/user/vendor/craftcms/cms/src/services/Globals.php(385): craft\services\Elements->saveElement(Object(craft\elements\GlobalSet), false)
  | #11 /app/user/vendor/craftcms/cms/src/base/ApplicationTrait.php(1669): craft\services\Globals->handleChangedGlobalSet(Object(craft\events\ConfigEvent))
  | #12 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(1116): craft\console\Application->craft\base\{closure}(Object(craft\events\ConfigEvent))
  | #13 [internal function]: craft\services\ProjectConfig->handleChangeEvent(Object(craft\events\ConfigEvent))
  | #14 /app/user/vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Array, Object(craft\events\ConfigEvent))
  | #15 /app/user/vendor/craftcms/cms/src/models/ProjectConfigData.php(79): yii\base\Component->trigger('addItem', Object(craft\events\ConfigEvent))
  | #16 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(1106): craft\models\ProjectConfigData->commitChanges(NULL, Array, 'globalSets.95c7...')
  | #17 [internal function]: craft\services\ProjectConfig->handleChangeEvent(Object(craft\events\ConfigEvent))
  | #18 /app/user/vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Array, Object(craft\events\ConfigEvent))
  | #19 /app/user/vendor/craftcms/cms/src/models/ProjectConfigData.php(79): yii\base\Component->trigger('addItem', Object(craft\events\ConfigEvent))
  | #20 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(1222): craft\models\ProjectConfigData->commitChanges(NULL, Array, 'globalSets.95c7...', true, NULL, true)
  | #21 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(1244): craft\services\ProjectConfig->craft\services\{closure}('globalSets.95c7...', true)
  | #22 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(568): craft\services\ProjectConfig->_applyChanges(Array, Object(craft\models\ProjectConfigData), Object(craft\models\ReadOnlyProjectConfigData))
  | #23 /app/user/vendor/craftcms/cms/src/migrations/Install.php(1133): craft\services\ProjectConfig->applyExternalChanges()
  | #24 /app/user/vendor/craftcms/cms/src/migrations/Install.php(78): craft\migrations\Install->insertDefaultData()
  | #25 /app/user/vendor/craftcms/cms/src/db/Migration.php(49): craft\migrations\Install->safeUp()
  | #26 /app/user/vendor/craftcms/cms/src/db/MigrationManager.php(233): craft\db\Migration->up(true)
  | #27 /app/user/vendor/craftcms/cms/src/console/controllers/InstallController.php(203): craft\db\MigrationManager->migrateUp(Object(craft\migrations\Install))
  | #28 [internal function]: craft\console\controllers\InstallController->actionCraft()
  | #29 /app/user/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
  | #30 /app/user/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
  | #31 /app/user/vendor/yiisoft/yii2/console/Controller.php(182): yii\base\Controller->runAction('', Array)
  | #32 /app/user/vendor/craftcms/cms/src/console/Controller.php(224): yii\console\Controller->runAction('', Array)
  | #33 /app/user/vendor/yiisoft/yii2/base/Module.php(552): craft\console\Controller->runAction('', Array)
  | #34 /app/user/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('install', Array)
  | #35 /app/user/vendor/craftcms/cms/src/console/Application.php(90): yii\console\Application->runAction('install', Array)
  | #36 /app/user/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('install', Array)
  | #37 /app/user/vendor/craftcms/cms/src/console/Application.php(121): yii\console\Application->handleRequest(Object(craft\console\Request))
  | #38 /app/user/vendor/yiisoft/yii2/base/Application.php(384): craft\console\Application->handleRequest(Object(craft\console\Request))
  | #39 /app/user/craft(24): yii\base\Application->run()
  | #40 {main}
  | *** failed to install Craft: An error occurred while executing the "craft\migrations\Install migration: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type integer: "f"
  | The SQL being executed was: SELECT "elementId"
  | FROM "structureelements"
  | WHERE ("structureId"=FALSE) AND ("elementId" IS NOT NULL)
  |  
  | Exception 'yii\db\Exception' with message 'SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "projectconfig" does not exist
  | LINE 1: DELETE FROM "projectconfig" WHERE "path"=$1
  | ^
  | The SQL being executed was: DELETE FROM "projectconfig" WHERE "path"='siteGroups.8ff16084-6771-4aaf-8aa4-89fcf15b9730.name''
  |  
  | in /app/user/vendor/yiisoft/yii2/db/Schema.php:676
  |  
  | Error Info:
  | Array
  | (
  | [0] => 42P01
  | [1] => 7
  | [2] => ERROR:  relation "projectconfig" does not exist
  | LINE 1: DELETE FROM "projectconfig" WHERE "path"=$1
  | ^
  | )


```

**Comment by ttempleton** [comment]
Thanks for reporting, should be fixed now in 3.1.4.

**Comment by boboldehampsink** [comment]
Thanks!
